databricks_cli/pipelines/cli.py

```diff
@@ -49,7 +49,7 @@
 
 
 @click.command(context_settings=CONTEXT_SETTINGS,
-               short_help='Deploys a delta pipeline according to the pipeline specification')
+               short_help='Deploys a pipeline according to the pipeline specification')
```

Comment:
All descriptions should use proper punctuation (for example, end a sentence with a period), here and throughout.

Reply:
done